README.md

```diff
@@ -196,6 +196,89 @@ answers = prompt(questions)
 ```
 
 The returned `answers` will be a dict containing the responses, e.g. `{"phone": "0123123", "continue": False, ""}`. The questions will be prompted one after another and `prompt` will return once all of them are answered.
+
+Each configuration dictionary needs to contain the following keys:
+
+* `'type'` - The type of the question.
+* `'name'` - The name of the question (will be used as key in the `answers` dictionary)
+* `'message'` - Message that will be shown to the user
+
+Optional Keys:
+
+* `'qmark'` - Question mark to use - defaults to `?`.
+* `'default'` - Preselected value.
+* `'choices'` - List of choices (applies when `'type': 'select'`).
+* `'when'` - Function checking if this question should be shown or skipped (same functionality than `.skip_if()`).
+* `'validate'` - Function or Validator Class performing validation (will be performed in real time as users type).
```

Comment:
Please have a close look at this list - these are the keys i was able to figure out myself as I needed these for my usecase - but it may be possible that i missed one or more of these i'm not aware about.